src/sentry/grouping/ingest.py

```diff
@@ -46,6 +47,19 @@
 Job = MutableMapping[str, Any]
 
 
+@dataclass
+class GroupHashInfo:
+    config: GroupingConfig
+    hashes: CalculatedHashes
+    grouphashes: list[GroupHash]
+    existing_grouphash: GroupHash | None
+
+
+NULL_GROUPING_CONFIG: GroupingConfig = {"id": "", "enhancements": ""}
+NULL_HASHES = CalculatedHashes(hashes=[], hierarchical_hashes=[], tree_labels=[])
+NULL_GROUPHASH_INFO = GroupHashInfo(NULL_GROUPING_CONFIG, NULL_HASHES, [], None)
+
```

Comment:
nit: IMO these belong in `sentry.grouping.api`

Reply:
Sure, I can move them there. 

At some point we should actually sit down and try to do a slightly better job of organizing all of the code in `sentry.grouping`, if nothing else to try to break up the super-long files into more manageable (and appropriately-named) ones.